app/services/journals/create_service.rb

```diff
@@ -901,6 +904,10 @@ def only_one_note(predecessor, notes)
       predecessor.notes.empty? || notes.empty?
     end
 
+    def unrestricted_journal?(restricted)
+      !restricted
+    end
```

Comment:
I think it would still be better to do this check as

```ruby
  predecessor.restricted == restricted
```

My thought is that if the previous and current updates are both restricted we should be okay aggregating them.